### PR TITLE
Add create Twitter Newsfeed component - Closes #913

### LINKED ISSUE #913: Create Newsfeed component - Parent #894
### Expected behaviour
Implemented newsFeed

Child of #894

### PR DESCRIPTION
**Pull request by michaeltomasik**
### What was the problem?
No Newsfeed component
### How did I fix it?
Implemented it
### How to test it?
- Go to dashboard
- You should see 
https://user-images.githubusercontent.com/1254342/41159896-58d2b99c-6b2e-11e8-9c13-fb4d9f1135bb.png
with login and without.
- Read me should open new tab with source of feed
- In Right corner there is `pencil icon` where you can select which feed to show
(Currentely there is only Twitter feed)

### Review checklist
- The PR solves #913 
- The PR follows our [Test guide](/LiskHQ/lisk-hub/blob/development/docs/TEST_GUIDE.md)
- The PR follows our [CSS guide](/LiskHQ/lisk-hub/blob/development/docs/CSS_GUIDE.md)

### CODE REVIEW

File: src/components/dashboard/index.js [hunk 1]
```diff
@@ -12,47 +12,57 @@ import CurrencyGraph from './currencyGraph';
...
+              transactions,
+              t,
+              address: account.address,
+              dashboard: true,
+              loading,
+              history,
+              onClick: props => history.push(`${routes.wallet.path}?id=${props.value.id}`),
+            }} />
+          </Box> :
+          <div></div>
```
Comment (faival) [inline src/components/dashboard/index.js]: null

File: src/components/newsFeed/news.js [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+  render() {
+    const {
+      source, content, timestamp, url, t,
+    } = this.props;
+
+    const hours = this.diffHours(new Date(), new Date(timestamp));
+    // hours, just now
+    let timestampMessage = hours ? `${hours}h ago` : 'just now';
+    // days
+    timestampMessage = hours > 24 ? `${Math.floor(hours / 24)} days ago` : timestampMessage;
```
Comment (faival) [inline src/components/newsFeed/news.js]: wouldn't ```hours % 24``` do the same?

File: src/components/newsFeed/news.js [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+    diff /= (60 * 60);
+    return Math.abs(Math.round(diff));
+  }
+
+  render() {
+    const {
+      source, content, timestamp, url, t,
+    } = this.props;
+
+    const hours = this.diffHours(new Date(), new Date(timestamp));
```
Comment (faival) [inline src/components/newsFeed/news.js]: I would either move to utils function or just use a 1 liner:
https://stackoverflow.com/questions/19225414/how-to-get-the-hours-difference-between-two-date-objects#19225540

File: src/components/newsFeed/news.js [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    // hours, just now
+    let timestampMessage = hours ? `${hours}h ago` : 'just now';
+    // days
+    timestampMessage = hours > 24 ? `${Math.floor(hours / 24)} days ago` : timestampMessage;
+
+    // Makes first letter capital
+    const sourceName = source.charAt(0).toUpperCase() + source.substr(1);
+    return (
+      <div className={`news ${styles.news}`}>
```
Comment (faival) [inline src/components/newsFeed/news.js]: I would change to `news-item`, in the tests might be easier to understand how many items are rendered.

File: src/components/newsFeed/newsFeed.js [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+          </div>
+          {this.state.showSettings ?
+            <form className={styles.form}>
+              <SettingsNewsFeed
+                t={this.props.t}
+                channels={this.props.channels}
+                hideSettings={this.hideSettings.bind(this)}
+                setNewsChannels={this.setNewsChannels.bind(this)} />
+            </form> :
+            <form className={styles.form}>
```
Comment (faival) [inline src/components/newsFeed/newsFeed.js]: Is the form html tag needed to show the list?

File: src/components/newsFeed/news.js (hunk 2 above)
Comment (michaeltomasik) [reply src/components/newsFeed/news.js]: yeaa `hours % 24 ` would also work but this one is more readable for me

**Comment by faival** [comment]
can we add more space in medium breakpoint to the readmore link?

image

**Comment by Efefefef** [comment]
@slaweet I need to have specification in #913 to test it
or does it have a parent issue?

**Comment by slaweet** [comment]
@Efefefef #913 has parent https://github.com/LiskHQ/lisk-hub/issues/894 as stated in its title.

**Comment by michaeltomasik** [comment]
@Efefefef I created a ticket for an empty newsfeed some time ago
https://github.com/LiskHQ/lisk-hub/issues/1153

**Comment by slaweet** [comment]
@michaeltomasik IMO it would be useful to use some tool that makes the links in posts clickable, like https://www.npmjs.com/package/react-format-text

**Comment by michaeltomasik** [comment]
According to what we were talking @vit I removed url links from description

**Pull request merged**